docs/product-getting-started-guide/src/main/asciidoc/task_process-forms.adoc

```diff
@@ -6,81 +6,143 @@ The following chapter instructs you how to create forms for your business proces
 [float]
 == Prerequisites
 
-* Finished business process model of the started project. For further information, see <<_modeling_a_business_process>>.
+Finished business process model of the started project. For further information, see <<_modeling_a_business_process>>.
 
 == Automatic Form Generation
 
-{PRODUCT} enables you to automatically generate all forms. To generate forms for your business process:
+{PRODUCT} enables you to automatically generate all forms. For this business process you are: Creating the Applicant form, Creating the Property form, and Creating the Application form.
```

Comment:
I'd suggest using bullet points for Creating the Applicant form, Creating the Property form, and Creating the Application form.